src/index.ts

```diff
@@ -1,64 +1,64 @@
-const SCHULD_2019 = 19.3 * 10 ** 9;
-const SCHULD_2022 = 24.3 * 10 ** 9;
-const SCHULD_DELTA = (SCHULD_2022 - SCHULD_2019) / (365 * 3 - 1) / 24 / 3600;
+const SCHULD_2019 = 19.3 * 10 ** 9
+const SCHULD_2022 = 24.3 * 10 ** 9
+const SCHULD_DELTA = ( SCHULD_2022 - SCHULD_2019 ) / ( 365 * 3 - 1 ) / 24 / 3600
 
 /*
 * Prijzen constants
 * */
-const PRIJS_FIETS = 150;
-const PRIJS_KRAT = 24;
-const PRIJS_BAPAO = 0.90;
+const PRIJS_FIETS = 150
+const PRIJS_KRAT = 24
+const PRIJS_BAPAO = 0.90
 
 const getSchuldToday = (): number => {
-    const today = new Date();
-    const secondsSince31Dec2019 = (today.getTime() - new Date('2019-12-31').getTime()) / 1000
-    return Math.round(SCHULD_2019 + SCHULD_DELTA * secondsSince31Dec2019);
+    const today = new Date()
+    const secondsSince31Dec2019 = ( today.getTime() - new Date( "2019-12-31" ).getTime() ) / 1000
+    return Math.round( SCHULD_2019 + SCHULD_DELTA * secondsSince31Dec2019 )
 }
 
 type Figures = {
-    fiets: number;
-    krat: number;
-    bapao: number;
+    fiets: number
+    krat: number
+    bapao: number
 }
 
-const getFigures = (schuld: number): Figures => {
+const getFigures = ( schuld: number ): Figures => {
     return {
-        fiets: Math.round(schuld / PRIJS_FIETS),
-        krat: Math.round(schuld / PRIJS_KRAT),
-        bapao: Math.round(schuld / PRIJS_BAPAO),
+        fiets: Math.round( schuld / PRIJS_FIETS ),
+        krat: Math.round( schuld / PRIJS_KRAT ),
+        bapao: Math.round( schuld / PRIJS_BAPAO ),
     }
 }
 
-const formatPrice = (amount: number): string => {
-    return Intl.NumberFormat('nl-NL', {
-        style: 'currency',
-        currency: 'EUR',
+const formatPrice = ( amount: number ): string => {
+    return Intl.NumberFormat( "nl-NL", {
+        style: "currency",
+        currency: "EUR",
         minimumFractionDigits: 0,
         maximumFractionDigits: 0,
-    }).format(amount);
+    } ).format( amount )
 }
-const formatNumber = (amount: number): string => {
-    return Intl.NumberFormat('nl-NL', {
+const formatNumber = ( amount: number ): string => {
+    return Intl.NumberFormat( "nl-NL", {
         minimumFractionDigits: 0,
         maximumFractionDigits: 0,
-    }).format(amount);
+    } ).format( amount )
 }
 
-window.addEventListener("load", () => {
-    const counter = document.getElementById('counter');
-    const counter_fiets = document.getElementById('counter_fiets');
-    const counter_krat = document.getElementById('counter_krat');
-    const counter_bapao = document.getElementById('counter_bapao');
+window.addEventListener( "load", () => {
+    const counter = document.getElementById( "counter" )
+    const counter_fiets = document.getElementById( "counter_fiets" )
+    const counter_krat = document.getElementById( "counter_krat" )
+    const counter_bapao = document.getElementById( "counter_bapao" )
```

Comment:
I'm not sure I like the double quotes and spaces before and after `()` t.b.h.

What code style does that adhere to?

Reply:
Of course it is a matter of preference, I would be happy to swap it out for no spaces and single quotes.